pkg/sbom/node_test.go

```diff
@@ -232,7 +232,7 @@ func TestNodeFlatString(t *testing.T) {
 					},
 				},
 			},
-			"bomsquad.protobom.Node.build_date:1700166898:bomsquad.protobom.Node.copyright:Copyright (c) 2023 The Protobom Authors:bomsquad.protobom.Node.file_name:test.tar.gz:bomsquad.protobom.Node.hashes:3:b51261db1ecadecf85274e811e537c5811a0ad1ab2a0121aeac4e3d031e1bf83:bomsquad.protobom.Node.id:node-id:bomsquad.protobom.Node.license_comments:Comment on the license:bomsquad.protobom.Node.license_concluded:Apache-2.0:bomsquad.protobom.Node.name:test node:bomsquad.protobom.Node.primary_purpose[0]:5:bomsquad.protobom.Node.release_date:1700080498:bomsquad.protobom.Node.url_download:http://example.com/file.tar.gz:bomsquad.protobom.Node.url_home:http://example.com/:bomsquad.protobom.Node.valid_until_date:1700253298:bomsquad.protobom.Node.version:1.0.0:extref:(t)56(u)http://github.com/external(c)GitHub Link:identifiers[1]:pkg:/apk/wolfi/bash@4.1.11:originator:n(The Open Source Authors)o(false)email(noone@example.com)url(http://github.com/example)p(987-5432):supplier:n(ACME, Inc)o(true)email(acme@example.com)url(http://acme-fixtures.com)p(123-3456)",
+			"extref:(t)56(u)http://github.com/external(c)GitHub Link:identifiers[1]:pkg:/apk/wolfi/bash@4.1.11:originator:n(The Open Source Authors)o(false)email(noone@example.com)url(http://github.com/example)p(987-5432):protobom.protobom.Node.build_date:1700166898:protobom.protobom.Node.copyright:Copyright (c) 2023 The Protobom Authors:protobom.protobom.Node.file_name:test.tar.gz:protobom.protobom.Node.hashes:3:b51261db1ecadecf85274e811e537c5811a0ad1ab2a0121aeac4e3d031e1bf83:protobom.protobom.Node.id:node-id:protobom.protobom.Node.license_comments:Comment on the license:protobom.protobom.Node.license_concluded:Apache-2.0:protobom.protobom.Node.name:test node:protobom.protobom.Node.primary_purpose[0]:5:protobom.protobom.Node.release_date:1700080498:protobom.protobom.Node.url_download:http://example.com/file.tar.gz:protobom.protobom.Node.url_home:http://example.com/:protobom.protobom.Node.valid_until_date:1700253298:protobom.protobom.Node.version:1.0.0:supplier:n(ACME, Inc)o(true)email(acme@example.com)url(http://acme-fixtures.com)p(123-3456)",
```

Comment:
@puerco this test required changing the ordering of the output to pass. I want to think that's expected by the alphabetical sorting but you should check me.